generator/src/lib.rs

```diff
@@ -1774,16 +1843,35 @@ fn generate_union(union: &vkxml::Union) -> Tokens {
         }
     }
 }
+pub fn root_create_info_names(definitions: &[&vkxml::DefinitionsElement]) -> HashSet<String> {
+    definitions
+        .iter()
+        .filter_map(|definition| match *definition {
+            vkxml::DefinitionsElement::Struct(ref _struct) => {
+                let is_root_create_info = _struct.extends.is_none();
```

Comment:
This is incorrect for `VkPhysicalDeviceProperties2`, which is both a root getter struct and an extension struct for `CreateInfo`

Reply:
Instead, why don't we use the set of things that appears in `structextends` attributes? I haven't seen any cases where those are layered, though maybe I haven't looked hard enough.

Reply:
> which is both a root getter struct and an extension struct for CreateInfo

What exactly do you mean by that? Afaik it is just used in `vkGetPhysicalDeviceProperties2`.

Reply:
Disregard, I was confused; too early for code review...

Reply:
Don't worry happens to me all the time :).